discovery-dns/src/main/resources/reference.conf

```diff
@@ -2,14 +2,19 @@
 # Akka Service Discovery DNS Config                  #
 ######################################################
 
-akka.discovery {
+akka {
+  discovery {
 
-  # Set the following in your application.conf if you want to use this discovery mechanism:
-  # impl = akka-dns
+    # Set the following in your application.conf if you want to use this discovery mechanism:
+    # impl = akka-dns
 
-  # configured the akka-dns provider
-  akka-dns {
-    # currently no special configuration is present for this module
-    class = akka.discovery.dns.DnsSimpleServiceDiscovery
+    # configured the akka-dns provider
+    akka-dns {
+      # currently no special configuration is present for this module
```

Comment:
Should this comment be deleted now that there is a `lookup-type` config below?

Reply:
Seen this @chbatey ?

Reply:
We've removed lookup-type now